bin/pulsar-client

```diff
@@ -19,7 +19,7 @@ BINDIR=$(dirname "$0")
 PULSAR_HOME=`cd $BINDIR/..;pwd`
 
 DEFAULT_CLIENT_CONF=$PULSAR_HOME/conf/client.conf
-DEFAULT_LOG_CONF=$PULSAR_HOME/conf/log4j.properties
+DEFAULT_LOG_CONF=$PULSAR_HOME/conf/log4j.shell.properties
```

Comment:
Yes, the `log4j.properties` is shared across all the components. So, as for the other tools (admin and perf) we are already using `log4j.shell.properties` to reduce the level to `WARN`